org.mlm.asteroids_revenge.yml

```diff
@@ -0,0 +1,30 @@
+app-id: org.mlm.asteroids_revenge
```

Comment:
This redirects to domainking, how do we verify you/project owns it? Can you put something up?

Reply:
Oh, i thought since it comes from my repo, so it says i own it. Is there any way to verify i own the project without paying for a website registar and redirect

Reply:
No you have to own the domain. If you cannot, you can use a `io.github.user.app` id and rename everything.

Reply:
i changed it to io.github.mlm.asteroids_revenge